From 4ae407a5a308febf63de27a62f8c301c73b37d3e Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Tue, 6 Oct 2009 17:44:33 +0200
Subject: [PATCH 1/5] Move config_init() after CreateWellKnownSockets() and InitCoreDevices()

config_init() can now add devices directly instead of scheduling a
timer.

Signed-off-by: Julien Cristau <jcristau@debian.org>
Tested-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 dix/main.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: xorg-server/dix/main.c
===================================================================
--- xorg-server.orig/dix/main.c
+++ xorg-server/dix/main.c
@@ -168,7 +168,6 @@
 	InitBlockAndWakeupHandlers();
 	/* Perform any operating system dependent initializations you'd like */
 	OsInit();
-        config_init();
 	if(serverGeneration == 1)
 	{
 	    CreateWellKnownSockets();
@@ -256,6 +255,7 @@
         InitCoreDevices();
 	InitInput(argc, argv);
 	InitAndStartDevices();
+        config_init();
 
 	dixSaveScreens(serverClient, SCREEN_SAVER_FORCER, ScreenSaverReset);
 
